tools/extract_tools.sh

```diff
@@ -27,23 +20,32 @@ git lfs ls-files | awk '{print $2}' | grep "-" &>/dev/null && {
   cd $DIR
 }
 
+LINARO_GCC=aarch64-linux-gnu-gcc
+GOOGLE_GCC_4_9=aarch64-linux-android-4.9
+EDK2_LLVM=llvm-arm-toolchain-ship
+SEC_IMAGE=SecImage
+
 LINARO_GCC_TARBALL=$LINARO_GCC.tar.gz
 GOOGLE_GCC_4_9_TARBALL=$GOOGLE_GCC_4_9.tar.gz
 EDK2_LLVM_TARBALL=$EDK2_LLVM.tar.gz
 SEC_IMAGE_TARBALL=$SEC_IMAGE.tar.gz
 
-if [ ! -d $LINARO_GCC ]; then
-  tar -xzf $LINARO_GCC_TARBALL &>/dev/null
-fi
+ARCH=$(uname -m)
+if [ "$ARCH" != "arm64" ] && [ "$ARCH" != "aarch64" ]; then
```

Comment:
why not use the same toolchains on arm64?

Reply:
tried it, doesn't work on arm64. works fine on x64. will test more and if I find a way will PR it separately.

keep in mind that previously was the same thing, I just moved the code around so that `extract_tools.sh` is called on every platform to check for git lfs. this toolchain was never used on arm64/aarch64.